packages/rancher-vsphere-cpi/charts/values.yaml

```diff
@@ -14,6 +14,11 @@ cloudControllerManager:
     repository: rancher/mirrored-cloud-provider-vsphere-cpi-release-manager
     tag: v1.2.1
   nodeSelector: {}
+  rbac:
+    enabled: true
+
+# Uncomment and set to "rke" or "rke2" to apply the node selector label appropriate for the cluster
+# clusterType: ""
```

Comment:
Should this line be un-commented? 

My concern is that whether the value will be set if the field does not exist in the value.yaml (since the field is commented out).

Reply:
It was, but I needed to comment it due to [this](https://github.com/rancher/dashboard/issues/3188#issuecomment-866588466).